OpenRA.Mods.Common/Traits/Render/SelectionDecorations.cs

```diff
@@ -33,17 +33,23 @@ public class SelectionDecorationsInfo : ITraitInfo, ISelectionDecorationsInfo
 
 		public readonly Color SelectionBoxColor = Color.White;
 
+		public readonly string Image = "pips";
+
 		public object Create(ActorInitializer init) { return new SelectionDecorations(init.Self, this); }
 
+		[Desc("Don't change the order of these sequences."), SequenceReference("Image")]
+		public readonly string[] PipSequences = { "pip-empty", "pip-green", "pip-yellow", "pip-red", "pip-gray", "pip-blue", "pip-ammo", "pip-ammoempty" };
+
+		[Desc("Don't change the order of these sequences."), SequenceReference("Image")]
+		public readonly string[] TagSequences = { "", "tag-fake", "tag-primary" };
+
+		// PipSequences and TagSequences depends on the order of pips in TraitsInterfaces.cs!
```

Comment:
> OpenRA.Mods.Common/Traits/Render/SelectionDecorations.cs:L46: [SA1512] A single-line comment must not be followed by a blank line. To ignore this error when commenting out a line of code, begin the comment with '////' rather than '//'.